source/src_lcao/FORCE_k.cpp

```diff
@@ -5,6 +5,7 @@
 #include <map>
 #include "../module_base/memory.h"
 #include "../module_base/timer.h"
+#include "wfc_dm_2d.h"  //caoyu add 2022-2-9
```

Comment:
No author and date info needed in code, for `git blame` is in charge of querying such info.